packages/react-components/lib/tasks/task-table-datagrid.tsx

```diff
@@ -313,6 +369,27 @@ export function TaskDataGridTable({
               default:
                 return classes.taskUnknownCell;
             }
+          } else if (params.field === 'unix_millis_finish_time') {
+            if (!params.value) {
+              return classes.taskUnknownCell;
+            }
+
+            const bookingLabel = getTaskBookingLabelFromTaskState(params.row);
```

Comment:
There are many excessive calls to `getTaskBookingLabelFromTaskState`, we can parse it once and pass it as the row params instead.

Reply:
https://github.com/open-rmf/rmf-web/pull/953/commits/a6797653fbd5cc56371d045a1ec5b8093d5ca372